rllib/policy/eager_tf_policy.py

```diff
@@ -516,8 +516,8 @@ def compute_actions(
                 _is_training=tf.constant(False),
             )
             if state_batches is not None:
-                for s in enumerate(state_batches):
-                    input_dict["state_in_{i}"] = s
+                for i, s in enumerate(state_batches):
```

Comment:
Note that this was a bug, I think, that is fixed by this change.

Reply:
Wow, nice catch!